web/satellite/src/components/accessGrants/CreateAccessModal.vue

```diff
@@ -0,0 +1,1028 @@
+            </div>
+            <div 
+                v-if="tooltipHover === 's3'" 
+                class="s3-tooltip"
+                @mouseover="toggleTooltipHover('s3','over')"
+                @mouseleave="toggleTooltipHover('s3','leave')"
+            >
+                <span class="tooltip-text">Generates access key, secret key, and endpoint to use in your S3-supporting application.  <a class="tooltip-link" href="https://docs.storj.io/dcs/api-reference/s3-compatible-gateway" target="_blank" rel="noreferrer noopener">Learn More</a></span>
+            </div>
+            <div 
+                v-if="tooltipHover === 'api'" 
+                class="api-tooltip"
+                @mouseover="toggleTooltipHover('api','over')"
+                @mouseleave="toggleTooltipHover('api','leave')"
+            >
+                <span class="tooltip-text">Creates access grant to run in the command line.  <a class="tooltip-link" href="https://docs.storj.io/dcs/getting-started/quickstart-uplink-cli/generate-access-grants-and-tokens/generate-a-token/" target="_blank" rel="noreferrer noopener">Learn More</a></span>
+            </div>
+            <!-- ********* Create Form Modal ********* -->
+            <form v-if="accessGrantStep === 'create'">
+                <div class="access-grant__modal-container__header-container">
+                    <h2 class="access-grant__modal-container__header-container__title">Create Access</h2>
+                    <div
+                        class="access-grant__modal-container__header-container__close-cross-container" @click="onCloseClick"
+                    >
+                        <CloseCrossIcon />
+                    </div>
+                </div>
+                <div class="access-grant__modal-container__body-container">
+                    <TypesIcon class="access-grant__modal-container__body-container__type-icon" />
+                    <div class="access-grant__modal-container__body-container__type">
+                        <p>Type</p>
+                        <div class="access-grant__modal-container__body-container__type__type-container">
+                            <input 
+                                id="acess-grant-check"
+                                v-model="checkedType"
+                                value="access" 
+                                type="radio"
+                                name="type" 
+                                :checked="checkedType === 'access'"
+                            >
+                            <label for="acess-grant-check">
+                                Access Grant
+                            </label>
+                            <img
+                                class="tooltip-icon"
+                                src="../../../static/images/accessGrants/create-access_information.png"
+                                @mouseover="toggleTooltipHover('access','over')"
```

Comment:
Is there a reason it's not using css to trigger the tooltip?

Reply:
Hi Egon, The reason is because the tooltip needs to stay open long enough when the icon is no longer hovered on for the user to click on the learn more link in the tooltip. Additionally this allows the tooltip to stay open when it is hovered over, and won't allow 2 to be open at once.